LayoutTests/imported/w3c/web-platform-tests/html/rendering/replaced-elements/attributes-for-embedded-content-and-images/video-intrinsic-width-height.html

```diff
@@ -21,7 +25,7 @@
         <!-- A width:height ratio other than 2:1 and overriding the specified style must be used to
              verify that width/height does not influence intrinsic ratio -->
         <video title="both width/height attributes and style"
-               data-expected-width="300" data-expected-height="150"
+               data-expected-width="300" data-expected-height="300"
```

Comment:
All browsers say 150, I think I'm going to revert this.
https://wpt.fyi/results/html/rendering/replaced-elements/attributes-for-embedded-content-and-images/video-intrinsic-width-height.html

Reply:
Ah, no, Firefox fails but isn't 150. I guess that's https://github.com/w3c/csswg-drafts/issues/10605

Reply:
Ah, you already reverted in https://github.com/WebKit/WebKit/pull/10386, but didn't export to WPT.